projects/defi-basket/index.js

```diff
@@ -59,6 +59,12 @@ function polygonTVL() {
               token.asset.type === "IAaveV2Deposit"
             ) {
               tokens.push(token.asset.address);
```

Comment:
can you edit this line for similar check as well, because if there is duplicate rows but `IAaveRewards` is first, we end up counting the token twice